htslib/hts.h

```diff
@@ -206,7 +206,7 @@ typedef struct htsFormat {
 struct __hts_idx_t;
 typedef struct __hts_idx_t hts_idx_t;
 
-typedef struct bam_hdr_t bam_hdr_t;
+struct bam_hdr_t;
```

Comment:
There's an (alphabetical) list of forward struct declarations further up (line 45ff) that this could be grouped with.